tools/create-element/main.ts

```diff
@@ -3,11 +3,14 @@ import { generateElement } from './generator/element.js';
 import Chalk from 'chalk';
 import Yargs from 'yargs';
 import prompts from 'prompts';
+import { readJson } from './generator/files.js';
+import { join } from 'node:path';
```

Comment:
what's this new magic?  `node:path`

Reply:
https://nodejs.org/dist/latest-v16.x/docs/api/esm.html#node-imports